Modules/binascii.c

```diff
@@ -334,14 +334,15 @@ binascii_a2b_uu_impl(PyObject *module, Py_buffer *data)
 binascii.b2a_uu
 
     data: Py_buffer
-    /
```

Comment:
AC currently seems doesn't support combining positional-only parameters with keyword-only parameters. So it's not intended but a side effect.